arbor/mechanisms/ProbAMPANMDA_EMS.mod

```diff
@@ -0,0 +1,329 @@
+COMMENT
```

Comment:
Let's not add these synapse mod files, until such time as we support them.
We don't need them for our benchmark.